src/vs-bicep/Bicep.VSLanguageServerClient.IntegrationTests/TestSolution/BicepTestProject/results/Formatting/BicepFormatting.bsl

```diff
@@ -0,0 +1,43 @@
+﻿param cosmosDBDatabaseName string = 'db1'
```

Comment:
Will testcase pass/fail be affected by the tab settings on the user's machine?

Reply:
Good question. I added a middle layer to use the default vscode settings as VS doesn't fully support settings/editorconfig yet. I created https://github.com/Azure/bicep/issues/7550 to handle the case

With regards to tests, it should use the default settings specified in middle layer.